### PR TITLE
php rule addition

### PR DESCRIPTION
**Pull request by ankit-privado**

### CODE REVIEW

File: rules/sinks/third_parties/sdk/league/php.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (HiteshMah-Jan) [inline rules/sinks/third_parties/sdk/league/php.yaml:1]: Not a third party

File: rules/sinks/third_parties/sdk/starkbank/php.yaml [hunk 2]
```diff
@@ -9,5 +9,5 @@ sinks:
     domains:
       - "starkbank.com"
     patterns:
-      - "(?i)(starkbank).*"
+      - "(?i)(starkbank|EllipticCurve).*"
```
Comment (HiteshMah-Jan) [inline rules/sinks/third_parties/sdk/starkbank/php.yaml:12]: ??

File: rules/sinks/third_parties/sdk/league/php.yaml (hunk 1 above)
Comment (ankit-privado) [reply rules/sinks/third_parties/sdk/league/php.yaml:1]: will remove.

File: rules/sinks/third_parties/sdk/starkbank/php.yaml (hunk 2 above)
Comment (ankit-privado) [reply rules/sinks/third_parties/sdk/starkbank/php.yaml]: removed it, as its a signing algo library.

**Review by HiteshMah-Jan — APPROVED** [review]

**Pull request merged**